src/mgmt/tests/mocks.c

```diff
@@ -493,6 +493,11 @@ ds_mgmt_destroy_pool(uuid_t pool_uuid, const char *group, uint32_t force)
 	return 0;
 }
 
+int
+ds_mgmt_evict_pool(uuid_t pool_uuid, const char *group)
```

Comment:
Should add implement the mock for this function and use it to verify ds_mgmt_drpc_evict_pool handles the return code correctly in a unit test